### PR TITLE
Feature/0.1.22+1

### PR DESCRIPTION
**Pull request by neeboo**
hot fix include:
https://github.com/AstroxNetwork/agent_dart/pull/19

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,37 +1,55 @@
+## Next
+
+- remove unused tests
+
+## 0.1.22+1
+
+-- ** 🩹 HotFix 🩹 ** remove assert on http
```
Comment (AlexV525) [inline CHANGELOG.md:7]:
```suggestion
- **🩹 HotFix 🩹** Remove invalid asserts `defaultFetch` (#19)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,37 +1,55 @@
+## Next
+
+- remove unused tests
+
+## 0.1.22+1
+
+- **🩹 HotFix 🩹** Remove invalid asserts `defaultFetch` (#19)
```
Comment (AlexV525) [inline CHANGELOG.md:7]:
```suggestion
- **🩹 HotFix 🩹** Remove invalid asserts with `defaultFetch` (#19)
```

**Review by AlexV525 — APPROVED** [review]

**Comment by AlexV525** [comment]
The commit tree seems not clean, but no conflicts were detected though.

**Comment by neeboo** [comment]
> The commit tree seems not clean, but no conflicts were detected though.

weird commit tree on my local, just found out

**Pull request merged**